engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/process/FlinkSource.scala

```diff
@@ -20,8 +31,23 @@ trait FlinkSource[T] extends Source[T] {
 
   def timestampAssigner : Option[TimestampAssigner[T]]
 
-}
+  override def startStream(env: StreamExecutionEnvironment, metaData: MetaData): DataStream[T] = {
```

Comment:
WDYT about rename it to `sourceStream`? WDYT about extracting metaData to implicit parameters - like it is in other places in our API?

Reply:
done :+1: